### PR TITLE
`config.yml` gains params for `2022Q4` data preparation

### PR DESCRIPTION
**Pull request by jdhoffa**

### CODE REVIEW

**Review by cjyetman — DISMISSED** [review]
I presume the scenarios have not been officially determined yet, so you're just leaving it to inherit from default for now?

**Comment by jdhoffa** [comment]
> I presume the scenarios have not been officially determined yet, so you're just leaving it to inherit from default for now?

Yes sorry good point, I will leave this as a draft until scenario list are finalized. 

I anticipate `weo_2022` and `geco_2022` (which are already available to pacta.scenario.preparation) will be included

File: config.yml [hunk 1]
```diff
@@ -78,3 +78,12 @@ default:
...
   global_aggregate_sector_list: ["Power"]
+
+2022Q4:
+  masterdata_ownership_filename: "2023-06-18_AI_RMI Bespoke_Company Data Products_masterdata_ownership_2022Q4.csv"
+  masterdata_debt_filename: "2023-06-18_AI_RMI Bespoke_Company Data Products_masterdata_debt_2022Q4.csv"
+  ar_company_id__factset_entity_id_filename: "2023-02-15_AI_RMI_Bespoke_Company_Data_Products_Company_ID_List_2022Q4.csv"
+  imf_quarter_timestamp: "2022-Q4"
+  factset_data_timestamp: "2022-12-31"
+  pacta_financial_timestamp: "2022Q4"
+  market_share_target_reference_year: 2022
```
Comment (cjyetman) [inline config.yml:89]: I thought the scenario sources were gonna change?

Comment (jdhoffa) [reply config.yml:89]: Sorry got my wires crossed and was looking at the `2022Q4_MFM` config branch. Will fix

**Review by cjyetman — APPROVED** [review]

**Pull request merged**